### DISCUSSION TITLE
How to disable spell check.

### DISCUSSION THREAD

**Discussion by shhivv** [mdx-editor/editor#351] [Ideas]
I want to disable the default spell check. Usually, I'd do this by setting `spellcheck="false"` in HTML, not sure how to do it here.

**Comment by petyosi** [comment]
Not supported currently, happy to accept a PR. I will move this to discussions, that's where I keep features like this.

**Comment by flamecommit** [comment]
I want this option, too

**Comment by Dari4sho** [comment]
First of all, thanks for building this! I love it so far. I was also in need of controlling the spell checking, so I added it real quick. It works fine for my local project -> https://github.com/mdx-editor/editor/pull/631

**Comment by petyosi** [comment]
Thanks to @Dari4sho who implemented the property.
